core/includes/theme.inc

```diff
@@ -2367,17 +2374,6 @@ function theme_item_list($variables) {
 }
 
 /**
- * Returns HTML for a help element.
- */
-function theme_help($variables) {
-  // Wrap in a paragraph tag if one is not present.
-  if (strpos($variables['markup'], '<p') === FALSE) {
-    $variables['markup'] = '<p>' . $variables['markup'] . '</p>';
-  }
-  return '<div class="help">' . $variables['markup'] . '</div>';
-}
-
-/**
```

Comment:
Oops, this looks like it may be a merge issue. We shouldn't be deleting `theme_help()`.